data/mods/Aftershock/recipes/bionic_recipes.json

```diff
@@ -345,7 +345,7 @@
   },
   {
     "type": "recipe",
-    "activity_level": "fake",
+    "activity_level": "LIGHT_EXERCISE",
```

Comment:
I'm thinking this should maybe be a moderate exercise?

Reply:
Same for all the bio armors

Reply:
Hammers and wrenches? Yeah, I agree. Moderate it is.